LayoutTests/TestExpectations

```diff
@@ -6271,3 +6271,16 @@ webkit.org/b/252183 imported/w3c/web-platform-tests/css/css-box/margin-trim/bloc
 imported/w3c/web-platform-tests/css/css-text/text-spacing/tentative/parsing/text-spacing-computed.html [ Skip ]
 imported/w3c/web-platform-tests/css/css-text/text-spacing/tentative/parsing/text-spacing-invalid.html [ Skip ]
 imported/w3c/web-platform-tests/css/css-text/text-spacing/tentative/parsing/text-spacing-valid.html [ Skip ]
+
+#Skipping scrollbar-color tests until the features get implemented.
+imported/w3c/web-platform-tests/css/css-scrollbars/transparent-on-root.html [ Skip ]
+imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar.html [ Skip ]
+imported/w3c/web-platform-tests/css/css-scrollbars/viewport-scrollbar-body.html [ Skip ]
+
+#These tests rely on dynamic scrollbar styling which isn't implemented yet.
+imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-001.html [ Skip ]
+imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-002.html [ Skip ]
+imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-width-paint-003.html [ Skip ]
+
+#This test relies on textarea resize handle showing without a scrollbar which isn't implemented yet.
+imported/w3c/web-platform-tests/css/css-scrollbars/textarea-scrollbar-width-none.html [ Skip ]
```

Comment:
Ideally these would refer to bugzilla bugs tracking their completion.

Reply:
Gonna have another look through this PR to try and fix the pipelines up today. Will also make or find bugs to link to for the skipped tests